common/lib/opentelemetry/common/utilities.rb

```diff
@@ -101,6 +101,36 @@ def cleanse_url(url)
         url
       end
 
+      # Returns the first non nil environment variable requested,
+      # or the default value if provided.
+      #
+      # @param [String] env_vars The environment variable(s) to retrieve
+      # @param default The fallback value to return if the requested
+      #  env var(s) are not present
+      #
+      # @returns [String]
+      def config_opt(*env_vars, default: nil)
+        env_vars.each do |env_var|
+          val = ENV[env_var]
+          return val unless val.nil?
+        end
+        default
```

Comment:
I am inclined to want to do this is a more functional way like this

```suggestion
        ENV.values_at(env_vars).compact.fetch(0, default)
```

However I am not certain if there is a guarantee that the Array returned from `values_at` will maintain the precedent order.

Reply:
🤷 this seems fine to me. 

It appears that the ordering should be considered predictable as the [method source](https://ruby-doc.org/core-2.7.1/Hash.html#method-i-values_at) shows it enumerating the provided args.
```
[9] pry(main)> ENV.values_at('LINES', 'COLUMNS')
=> ["37", "181"]
[10] pry(main)> ENV.values_at('COLUMNS', 'LINES')
=> ["181", "37"]
```